common/interfaces/user_interface/ui_multi_instruction.c

```diff
@@ -383,3 +406,37 @@ void multi_instruction_init(const char **arr, const uint8_t count, const uint16_
 
     multi_instruction_create();
 }
+
+void multi_instruction_with_image_init(instruction_content_t content[], const uint8_t count, const uint16_t delay_in_ms, const bool destruct_on_click)
+{
+
+    data = NULL;
+    data = malloc(sizeof(struct Multi_Instruction_Data));
```

Comment:
Add assertions to ensure `data` & `obj` variables are indeed allocated before dereferencing them in subsequent statements

Reply:
added